exonum-java-binding/core/src/main/java/com/exonum/binding/core/blockchain/Block.java

```diff
@@ -90,6 +93,19 @@ public final boolean isEmpty() {
    */
   public abstract HashCode getStateHash();
 
+  /**
+   * Root hash of exceptions occurred in the block. Empty if no errors.
```

Comment:
> Empty if no errors.

Neither Block documentation nor protobuf says it can be empty.

Reply:
Thanks, found special value for this case `7324b5c72b51bb5d4c180f1109cfd347b60473882145841c39f3e584576296f9`